salt/profiles/config/etc-init-uwsgi-profiles.conf

```diff
@@ -2,4 +2,9 @@ description "uWSGI server"
 start on (local-filesystems and net-device-up IFACE!=lo)
 respawn
 env LANG=en_US.UTF-8
+{% if pillar.elife.newrelic.enabled %}
+env NEW_RELIC_CONFIG_FILE=/srv/profiles/newrelic.ini
+exec /srv/profiles/venv/bin/newrelic-admin run-program /srv/profiles/venv/bin/uwsgi --enable-threads --die-on-term --ini /srv/profiles/uwsgi.ini
```

Comment:
Copied addition of `--enable-threads`, meant?

Reply:
New Relic uses a background thread to upload data to their servers, so it's needed. But it's not needed if there is no New Relic, so it doesn't go into general uwsgi configuration.

Reply:
💁‍♂️